packages/react/src/themes/types.ts

```diff
@@ -213,6 +238,7 @@ export interface ComponentSlotStylesPrepared<TProps = {}, TVars = {}>
   extends ObjectOf<ComponentSlotStyleFunction<TProps, TVars>> {}
 
 export interface ComponentSlotClasses extends ObjectOf<string> {}
+
```

Comment:
I don't understand this change but would definitely remove the export above or below :-)

Reply:
I think it was suppose to be just empty row, but I see now that we have the same line twice:
`export interface ComponentSlotClasses extends ObjectOf<string> {}`
Will remove one of them...

Reply:
@miroslavstastny do you agree?

Reply:
Yup, that's what I've meant.